### DISCUSSION TITLE
Show calendar with multiple categories

### DISCUSSION THREAD

**Discussion by MichaelWybo** [liedekef/events-made-easy#652] [General]
Hello Franky

As a person can  book events of multiple categories, I try to show a calendar with all events of the categories which these person have access:

i tried

[eme_calendar category=#_GROUPS  full=1] but it's not working

any idea ?

Alvast bedankt

**Comment by liedekef** [comment]
eme_calendar is not a people-context shortcode, so #_GROUPS doesn't get replaced (and it would need to get replaced before the rest is checked).
However, in your case the solution can be achieved with the current EME version:
- put your shortcode in a template
- put [eme_person template_id=xx] (with xx the id of the created template) on a page. That will cause the template to be fetched, #_GROUPS to be replaced and then the shortcode in the template to be executed.

**Reply by MichaelWybo** [reply]
Hey Franky 

I see where I'm faulty,

Events are linked to categories
Persons arte linked to groups

When I setup EME few years agoI I tried to link events to groups of persons and to do so I gave the the same name to groups and categories.

#_GROUPS is retrieving group's name
the "category=" filter in the shortcode is working with a single group's name, not multiple, or ID

**_EDIT_** it's working with this shortcode [eme_calendar category="#_GROUPS" full=1] so dont't forget the quotes :)
